distribution/changelog.txt

```diff
@@ -10,12 +10,14 @@
 - Improved: [#18945] Languages can now fall back to other languages than English.
 - Improved: [#18970] Trying to load a non-park save will now display a context error.
 - Improved: [#19044] Added special thanks to RMC and Wiegand to the About page.
+- Improved: [#19067] New Ride window now allows filtering similarly to Object Selection.
 - Change: [#19018] Renamed actions to fit the naming scheme.
 - Fix: [#18467] “Selected only” Object Selection filter is active in Track Designs Manager, and cannot be toggled.
 - Fix: [#18905] Ride Construction window theme is not applied correctly.
 - Fix: [#18911] Mini Golf station does not draw correctly from all angles.
 - Fix: [#18971] New Game does not prompt for save before quitting.
 - Fix: [#19026] Park loan is clamped to a 32-bit integer.
+- Fix: [#19067] Object selection filter is updated correctly when set to empty string.
```

Comment:
When you move this fix to its own PR (see the other line note), please reword. The changelog and commit message should list the problem that was solved, rather than the solution.

Reply:
This has been removed along with the corresponding code change